ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-master.yaml

```diff
@@ -95,7 +95,7 @@ tests:
           cd /work && \
           entrypoint \
           prow/integ-suite-ocp.sh \
-          pilot 'TestGatewayConformance|TestCustomGateway|TestCNIRaceRepair|TestGateway/managed-owner|TestProxyHeaders|TestCustomGateway/helm|TestCustomGateway/helm-simple|TestTcpProbe/rewrite-success|TestGRPCProbe|TestTunnelingOutboundTraffic|TestCNIVersionSkew|TestGateway|TestIngress|TestTraffic/jwt-claim-route'"
+          pilot 'TestGatewayConformance|TestCustomGateway|TestCNIRaceRepair|TestGateway/managed-owner|TestProxyHeaders|TestCustomGateway/helm|TestCustomGateway/helm-simple|TestTcpProbe/rewrite-success|TestGRPCProbe|TestTunnelingOutboundTraffic|TestCNIVersionSkew|TestGateway|TestIngress|TestTraffic/jwt-claim-route|TestLabelChanges|TestLocality|TestMirroring/mirror-percent-absent|TestMirroring/mirror-50|TestMirroring/mirror-10|TestMirroringExternalService|TestTraffic/virtualservice|TestTraffic/serverfirst/tcp-server|TestTraffic/loop|TestTraffic/tls-origination|TestTraffic/externalname|TestTraffic/host|TestTraffic/envoyfilter|TestTraffic/consistent-hash|TestTraffic/externalservice|TestTraffic/upstreamproxy'"
```

Comment:
Could you please add a comment with a jira for fixing these tests and a note to revert this when the jira is fixed? Se we don't forget it here.

Reply:
Sure. I'll update the jiras and add the notes

Reply:
I just added the information @FilipB

Reply:
Wait, make update deleted the comments

Reply:
Ok, now is Ok, now the comments are added @FilipB